tutorials/s3-to-hdfs-sync/src/site/conf/cluster-memory-conf.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+
```

Comment:
@ashwinchandrap Could you please explain why are we specifying operator properties in different property file.